docs-src/typescript/index.md

```diff
@@ -11,7 +11,7 @@ tags:
 
 [![NPM](https://img.shields.io/npm/v/temporalio.svg?style=for-the-badge)](https://www.npmjs.com/search?q=author%3Atemporal-sdk-team)
 
-This project requires Node.js 14 or later.
+This project requires Node.js 14.18 or later.
 
 **Create a new project**
```

Comment:
```suggestion
**Create a project**
```
Yeah, technically out of scope for this PR, but in reach of my mouse. Changed per https://developers.google.com/style/word-list#create-new.